src/MainnetController.sol

```diff
@@ -274,6 +286,72 @@ contract MainnetController is AccessControl {
         );
     }
 
+    /**********************************************************************************************/
+    /*** wstETH Integration                                                                     ***/
+    /**********************************************************************************************/
+
+    function depositToWstETH(uint256 amount) external {
+        _checkRole(RELAYER);
+        _rateLimitedAsset(LIMIT_WSTETH_DEPOSIT, Ethereum.WETH, amount);
```

Comment:
This doesn't need to be hashed against the asset cause wstETH and WETH will always be used